securitas/themes/default/static/css/default.css

```diff
@@ -1 +1,9 @@
 /*CSS rules for the 'default' securitas theme */
+
+html, body {
+  height: 100vh;
+  max-width: 100vw;
+  margin: 0;
+  display: flex;
+  flex-flow: column;
+}
```

Comment:
What are these rules fixing here? Is this something the stock bootstrap doesnt give us?

Reply:
Eh, frankly this would work as well in the shared css file, it's only here because this bit is different for the Fedora theme, but it shouldn't matter in the end